### DISCUSSION TITLE
Editable ComboBox

### DISCUSSION THREAD

**Discussion by VHSgunzo** [slint-ui/slint#3955] [Q&A]
Hi, is it possible to make an editable ComboBox?

**Comment by ogoffart** [comment]
Currently, there are no builtin widgets to do editable ComboBox.

So the way to do that would be to use a LineEdit and a PopupWindow.
There is an example of how to make a custom combobox there: https://github.com/slint-ui/slint/blob/037e5e462e1711a09e944ab5e9839cb485a0d992/examples/printerdemo/ui/common.slint#L170
